test/schemes/boundary/open_boundary/pressure_model.jl

```diff
@@ -0,0 +1,160 @@
+
+            signal = 0.0
+            for (k, c) in zip(-10:10, c_q)
+                signal += real(c * exp(1im * k * 2 * pi / T * t))
+            end
+
+            return signal
+        end
+
+        # Mock fluid system
+        struct FluidSystemMockRCR <: TrixiParticles.AbstractFluidSystem{2}
+            pressure_acceleration_formulation::Nothing
+            density_diffusion::Nothing
+        end
+        TrixiParticles.initial_smoothing_length(system::FluidSystemMockRCR) = 1.0
+        TrixiParticles.nparticles(system::FluidSystemMockRCR) = 1
+        TrixiParticles.system_smoothing_kernel(system::FluidSystemMockRCR) = nothing
+
+        # Encapsulated simulation updating only `calculate_flow_rate_and_pressure!`
+        function simulate_rcr(R1, R2, C, tspan, dt, p_0, func)
+            pressure_model = RCRWindkesselModel(; peripheral_resistance=R2,
+                                                compliance=C,
+                                                characteristic_resistance=R1)
+            # Define a boundary zone with height=1.0 to ensure a unit volume,
+            # so velocity directly corresponds to flow rate.
+            reference_velocity(pos, t) = SVector(func(t), 0.0)
+            boundary_zone = BoundaryZone(; boundary_face=([0.0, 0.0], [0.0, 1.0]),
+                                         particle_spacing=0.1, face_normal=(-1.0, 0.0),
+                                         density=1000.0, reference_velocity,
+                                         reference_pressure=pressure_model,
+                                         open_boundary_layers=1, rest_pressure=p_0)
+
+            system = OpenBoundarySystem(boundary_zone; buffer_size=0,
+                                        boundary_model=nothing,
+                                        fluid_system=FluidSystemMockRCR(nothing, nothing))
+            system.boundary_zone_indices .= 1
+
+            u = system.initial_condition.coordinates
+            v = system.initial_condition.velocity
+
+            times = collect(tspan[1]:dt:tspan[2])
+            p_calculated = empty(times)
+            for t in times
+                v[1, :] .= func(t)
+                TrixiParticles.calculate_flow_rate_and_pressure!(system, v, u, dt)
+
+                # Store only values after the seventh cycle
+                if t >= 7T
+                    p = TrixiParticles.reference_pressure(boundary_zone, v, system, 1, 0, t)
+                    push!(p_calculated, p)
+                end
+            end
+
+            return p_calculated
+        end
+
+        T = 0.375
+        dt = T / 500
+        tspan = (0.0, 8T)
+        p0 = 78.0
+
+        R1 = 1.7714
+        R2 = 106.66
+        C = 1.1808e-2
+
+        # The reference pressure values are computed using an ODE that describes
+        # the behavior of the RCR Windkessel model. The governing equation is:
+        #
+        #   dp/dt + p / (R_1 * C) = R_2 * dq/dt + (R_1 + R_2) / (R_1 * C) * q
```

Comment:
```suggestion
        #   dp/dt + p / (R_2 * C) = R_2 * dq/dt + (R_1 + R_2) / (R_1 * C) * q
```

Below you use it with R_2